meta/documentation_style.rst

```diff
@@ -185,3 +185,10 @@ The subheadings which are always deemed as client-affecting are:
 * Redactions
 
 When a new subheading is added, it must be referenced and ordered in this document.
+
+Describing grammar
+~~~~~~~~~~~~~~~~~~
+
+Use  [RFC5234-style ABNF](https://datatracker.ietf.org/doc/html/rfc5234) when describing
```

Comment:
this is not how you write an RST link

```suggestion
Use  `RFC5234-style ABNF <https://datatracker.ietf.org/doc/html/rfc5234>`_ when describing
```

I *think*.

Reply:
Argh, I remembered it was RST for the header and then immediately forgot.

Reply:
(github preview says yes, that's an RST link)